corehq/apps/app_manager/static/app_manager/js/download_index_main.js

```diff
@@ -21,6 +21,13 @@ hqDefine('app_manager/js/download_index_main',[
                 initialPageData.reverse("get_multimedia_sizes", appID, 'default'));
             multimediaSizes.load();
             $("#multimedia-sizes-container").koApplyBindings(multimediaSizes);
+            if ($('#build-profile-select-for-multimedia').length) {
+                $('#build-profile-select-for-multimedia').on('change', function() {
```

Comment:
Mixing jQuery manipulation and knockout views makes code harder to reason about. Can you expand the scope of `multimediaSizesView` to include the build profile? I'd imagine renaming the current `multimediaSizesView` to something like `multimediaSizesForApp`, and then having an outer `multimediaSizesView` that encompasses the build profile id and either one or two `multimediaSizesForApp` views, depending on whether you're ono View Source Files or the build diff page.

Reply:
Are you suggesting
1. move this logic around deciding the url based on build profile id in the current view?
2. KO binding the view to the select dropdown section and handle changes to build profile, hence everything in that view?

Reply:
#2

Reply:
That was fun, updated
https://github.com/dimagi/commcare-hq/pull/26032/commits/b0d594acdc054b3ebb1dd3e64f0567d4419af295
for just setup

and then
https://github.com/dimagi/commcare-hq/pull/26032/commits/e20866e8178b1c8cd7e5652173b141d6c525cdd2